openmdao/docs/feature_reference/working_with_models/debugging/om_command.rst

```diff
@@ -0,0 +1,120 @@
+.. _om-command:
+
+******************
+Command Line Tools
+******************
+
+OpenMDAO has a number of debugging/viewing command line tools that are available via the `om`
+command.  There are two types of commands available, those that perform some sort of viewing or
+configuration checking on the Problem after its setup is complete, and those that are used to
+collect information about the entire run of the Problem, things like profilers and tracers.
+
+
+Post-setup Commands
+-------------------
+
+The following commands all run immediately after a Problem's final_setup function.  After they
+complete, the program will exit, rather than continuing to the end of the user's run script.
+This makes it convenient to view or check the configuration of a model in any run script without
+having to wait around for the entire script to run.
+
```

Comment:
I guess I'll have to be the one that says `om` seems like it might be overly terse.  In previous iterations we had an `openmdao` command which didn't seem all that cumbersome and was less likely to collide with other applications (a quick search finds Open Music and Operations Manager using the `om` abbreviation). The user could easily alias to the shorter version if he/she has a posix bent and no competition for the name.

Reply:
om was suggested by Jubs and I thought it was ok. I'm also ok with openmdao though.